### PR TITLE
Update (most) old fluo-io links

### PR DESCRIPTION
**Pull request by ctubbsii**

### CODE REVIEW

File: _posts/blog/2014-12-30-stress-test-long-run.md [hunk 1]
```diff
@@ -22,19 +22,19 @@ everything finished, the test was a success. The number of unique integers
...
-     $ java -cp $STRESS_JAR io.fluo.stress.trie.Print $FLUO_PROPS
+     $ java -cp $STRESS_JAR stresso.trie.Print $FLUO_PROPS
      Total at root : 1011489250
      Nodes Scanned : 59605
 
 Below are a few lines of output selected from the map reduce job.
 
      Map input records=1011999273
-     io.fluo.stress.trie.Unique$Stats 
+     stresso.trie.Unique$Stats
```
Comment (keith-turner) [inline _posts/blog/2014-12-30-stress-test-long-run.md]: this is not what the code printed at that time

File: _posts/release/2016-01-12-fluo-1.0.0-beta-2.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ made.  The following API changes happened between beta-1 and beta-2.
 ## Testing
 
 For this release, a long run of the [Webindex][webindex] application was performed and is documented in
-[a blog post][webindex-post].  A long run of [Fluo stress][fluo-stress] was run and documented in 
+[a blog post][webindex-post].  A long run of [stresso][stresso] was run and documented in
```
Comment (keith-turner) [inline _posts/release/2016-01-12-fluo-1.0.0-beta-2.md]: could capatilize stresso for consistency

File: docs/fluo/1.0.0-alpha-1/stress.md [hunk 3]
```diff
@@ -22,10 +22,10 @@ These tests can be run using `mvn verify`.
 Run trie stress test on cluster
 -------------------------------
 
-If you want to run the trie stress on the cluster, first set up HDFS, YARN, Zookeeper, 
+If you want to run the trie stress on the cluster, first set up HDFS, YARN, Zookeeper,
 and Accumulo. Next, initialize Fluo with following observer:
 
-    io.fluo.observer.0=io.fluo.stress.trie.NodeObserver
+    io.fluo.observer.0=stresso.trie.NodeObserver
```
Comment (keith-turner) [inline docs/fluo/1.0.0-alpha-1/stress.md]: not the package name at the time

File: docs/fluo/1.0.0-alpha-1/stress.md [hunk 4]
```diff
@@ -34,22 +34,22 @@ Next, build the module:
...
-    cp target/fluo-stress-1.0.0-alpha-1-SNAPSHOT.jar $DEPLOY/lib/observers
+    cp target/stresso-1.0.0-alpha-1-SNAPSHOT.jar $DEPLOY/lib/observers
 
-Finally, on a node where Hadoop is set up, run the following command to ingest 
-data into Fluo using fluo-stress-1.0.0-alpha-1-SNAPSHOT-jar-with-dependencies.jar:
+Finally, on a node where Hadoop is set up, run the following command to ingest
+data into Fluo using stresso-1.0.0-alpha-1-SNAPSHOT-jar-with-dependencies.jar:
 
-    yarn jar <jarPath> io.fluo.stress.trie.NumberIngest <numMap> <numPerMap> <nodeSize> <fluoProps>
+    yarn jar <jarPath> stresso.trie.NumberIngest <numMap> <numPerMap> <nodeSize> <fluoProps>
```
Comment (keith-turner) [inline docs/fluo/1.0.0-alpha-1/stress.md]: not the package name for at alpha time

File: resources/blog/stress_1/fluo.properties [hunk 5]
```diff
@@ -62,7 +62,7 @@ io.fluo.admin.accumulo.classpath=${io.fluo.admin.hdfs}/fluo/lib/fluo-api-1.0.0-b
 # io.fluo.observer.0=com.foo.Observer1
 # Can optionally have configuration key values
 # io.fluo.observer.1=com.foo.Observer2,configKey1=configVal1,configKey2=configVal2
-io.fluo.observer.0=io.fluo.stress.trie.NodeObserver
+io.fluo.observer.0=stresso.trie.NodeObserver
```
Comment (keith-turner) [inline resources/blog/stress_1/fluo.properties]: another package name time traveler

File: _posts/blog/2014-12-30-stress-test-long-run.md [hunk 6]
```diff
@@ -22,19 +22,19 @@ everything finished, the test was a success. The number of unique integers
 computed independently by MapReduce matched the number computed by Fluo. Below
 is the output of the stress test count from Fluo.
 
-     $ java -cp $STRESS_JAR io.fluo.stress.trie.Print $FLUO_PROPS
+     $ java -cp $STRESS_JAR stresso.trie.Print $FLUO_PROPS
```
Comment (keith-turner) [inline _posts/blog/2014-12-30-stress-test-long-run.md]: not the package name at the time

Comment (mikewalch) [reply _posts/blog/2014-12-30-stress-test-long-run.md]: I agree with Keith. Blog posts should be updated only in special situations (i.e to fix a broken link or correct wrong information).  I don't think blog posts should be updated due to code changes in later Fluo releases.

File: _posts/release/2015-06-09-fluo-1.0.0-beta-1.md [hunk 7]
```diff
@@ -31,18 +31,18 @@ This release contains many new features that makes it easier to run, develop, an
 #### Simplified Fluo administration on a local machine or EC2 cluster
 
 Developers can now run Fluo and its dependencies on their local machine ([#92][92]) or an AWS EC2 cluster ([#356][356]) using a few simple commands.
-This was done by creating two administration tools called [Fluo-dev][fluo-dev] and [Fluo-deploy][fluo-deploy] whose scripts and configuration reside in repos
+This was done by creating two administration tools called [Uno] and [Muchos] whose scripts and configuration reside in repos
```
Comment (mikewalch) [inline _posts/release/2015-06-09-fluo-1.0.0-beta-1.md:34]: I like this change as it does reduce confusion.

File: _posts/blog/2014-12-30-stress-test-long-run.md (hunk 6 above)
Comment (ctubbsii) [reply _posts/blog/2014-12-30-stress-test-long-run.md]: Some of the find/replace was a bit aggressive. I'll fix any other errors you guys find like this.

**Comment by ctubbsii** [comment]
This PR is way out of date. Closing. Can re-attempt this another time.

**Pull request closed without merging**